### ISSUE TITLE
Spock blocks when using tango 9.3

### ISSUE THREAD

**Issue by tiagocoutinho** [sardana-org/sardana#1152]
Tested with the following conda environment:
```
PyTango 9.3.0 (9, 3, 0)
PyTango compiled with:
    Python : 2.7.16
    Numpy  : 1.11.3
    Tango  : 9.3.2
    Boost  : 1.67.0

PyTango runtime is:
    Python : 2.7.16
    Numpy  : 1.16.3
    Tango  : 9.3.2
    Boost  : 0.0.0

Sardana: 2.7.3-alpha

taurus: 4.5.4-alpha
```
(Pytango and tango from conda, taurus and sardana from latest develop branch on github)

To reproduce the problem install the above versions (critical are pytango and tango), launch a sardana server and a spock client.
Typing `lsm` in spock and the console is blocked indefinitely.

The same with PyTango 9.2.5 and Tango C++ 9.2.5 works fine.

**Comment by reszelaz** [comment]
This issue does not only affect Sardana but any Tango spectrum attriubute of DevString type pushing events. These events does not arrive to the client.

The good news is that this is already solved in PyTango 9.3.1 (at least in commit 9cf78c):

```
PyTango 9.3.1 (9, 3, 1, 'dev', 0)
PyTango compiled with:
    Python : 3.5.3
    Numpy  : 1.12.1
    Tango  : 9.3.3
    Boost  : 1.62.0

PyTango runtime is:
    Python : 3.5.3
    Numpy  : 1.12.1
    Tango  : 9.3.3
    Boost  : 0.0.0

```

**Comment by reszelaz** [comment]
I close this issue. Either use the previous versions or update the Tango stack.
